.github/workflows/testAndPublish.yml

```
@@ -409,7 +420,7 @@ jobs:
       contents: write
       deployments: write
     runs-on: ubuntu-latest
-    needs: [buildNVDA, checkPo, checkPot, licenseCheck, unitTests, createLauncher, systemTests, createSymbols]
+    needs: [buildNVDA, createLauncher, createSymbols, allTestsPass]
```

Aren't `buildNVDA`, `createLauncher` and `createSymbols` redundant?
```suggestion
    needs: [allTestsPass]
```

---

no they need to be specified as we use `needs.buildLauncher` etc below

---

Aah right, thanks.